boards/shields/nrf7002ek/Kconfig.shield

```diff
@@ -7,17 +7,17 @@
 DT_COMPAT_NORDIC_NRF700X_QSPI := nordic,nrf700x-qspi
 DT_COMPAT_NORDIC_NRF700X_SPI := nordic,nrf700x-spi
 
-config SHIELD_NRF7002EK_NRF7002
-	def_bool $(shields_list_contains,nrf7002ek_nrf7002)
+config SHIELD_NRF7002EK
+	def_bool $(shields_list_contains,nrf7002ek)
 
 config NRF7002_ON_QSPI
 	def_bool $(dt_compat_enabled,$(DT_COMPAT_NORDIC_NRF700X_QSPI))
-	depends on SHIELD_NRF7002EK_NRF7002
+	depends on SHIELD_NRF7002EK
 
 config NRF7002_ON_SPI
 	def_bool $(dt_compat_enabled,$(DT_COMPAT_NORDIC_NRF700X_SPI))
-	depends on SHIELD_NRF7002EK_NRF7002
+	depends on SHIELD_NRF7002EK
 
 config BOARD_NRF7002
```

Comment:
What's the purpose of this option actually?

Reply:
It is redundant now. Removed it in latest push.